Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -130,21 +130,21 @@ public let CodeGenerators: [CodeGenerator] = [
 
     CodeGenerator("PlainFunctionGenerator") { b in
         b.definePlainFunction(withSignature: FunctionSignature(withParameterCount: Int.random(in: 2...5), hasRestParam: probability(0.1)), isStrict: probability(0.1)) { _ in
-            b.generateRecursive()
+            b.generateRecursive(mode: .codeGenOnly)
```

Comment:
I wonder if it would be better to have a `currentCodeGenMode` in the ProgramBuilder (I guess set upon entry in `generate` and reset to some "None" value when the function returns) so that `generateRecursive` just uses that mode. This way we wouldn't need to specify the mode all the time in this file, and it would also allow us to do proper `CodeGenAndSplice` code generation. WDYT?